styles/pages/Home.module.scss

```diff
@@ -419,37 +433,44 @@
     flex-direction: column;
     padding: 48px 16px 48px 16px;
     border-radius: 8px;
+    align-items: center;
+    align-content: center;
+
     color: $primary-black-1;
-    background-color: #f5f5f5;
+    background-color: #ffffff;
 
     .statRow {
-      width: 800px;
+      width: 1200px;
+      display: flex;
+      flex-direction: row;
+
     }
 
     .statColLeft {
```

Comment:
Can rename this class, since there's no left and right anymore

Reply:
Resolved in commit 2262237a957aa871503cb28f631e659323cbee28